### ISSUE TITLE
confused config of kapacitor.db

### ISSUE THREAD

**Issue by sfwn** [influxdata/influxdata-docker#56]
kapacitor docker image version: **1.0-alpine**

I run kapacitor through docker, volume `/var/lib/kapacitor/:/var/lib/kapacitor/` for data persistence.
But after I stop & remove kapacitor-docker-container, recreate & run this container again, I lost all my tasks by typing `# kapacitor list tasks` inside container.

Inside container, I found default config file `/etc/kapacitor/kapacitor.conf` with content below:
```
data_dir = "/var/lib/kapacitor"

[replay]
  dir = "/var/lib/kapacitor/replay"

[storage]
  dir = "/var/lib/kapacitor/kapacitor.db"
```

But I found the real **kapacitor.db** is under the root dir: `/kapacitor.db` and when I run docker container with `-v /var/lib/kapacitor/kapacitor.db:/kapacitor.db`, the tasks persist.

I am confused with `/kapacitor.conf` and `[storage]` config in `/etc/kapacitor/kapacitor.conf`.

**Comment by sfwn** [comment]
how can I persist data?

**Comment by nathanielc** [comment]
The config should read:

```
[storage]
  boltdb = "/var/lib/kapacitor/kapacitor.db"
```

Looks like the `1.0` default config is wrong. It has been fixed for 1.1 and 1.2 already.
